### PR TITLE
fix: modal tailwindcss config

### PR DESCRIPTION
**Pull request by winchesHe**
## 📝 Description

closes : #99 

## ✅ Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Refactoring (improve a current implementation without adding a new feature or fixing a bug)
- [ ] Improvement (non-breaking change which improves an existing feature)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update

### CODE REVIEW

**Comment by winchesHe** [comment]
fixed in the file `src/constants/required.ts`

File: src/helpers/debug.ts [hunk 1]
```diff
@@ -12,4 +14,41 @@ export async function debugExecAddAction(cmd: string, components: string[] = [])
   } else {
     await exec(cmd);
   }
+}
+
+export function debugAddedPkg(components: string[], packagePath: string) {
+  if (!components.length) return
+  if (getStoreSync('debug')) {
```
Comment (wingkwong) [inline src/helpers/debug.ts:21]:
```ts
if (!components.length || !getStoreSync('debug')) return
```

File: src/helpers/debug.ts [hunk 2]
```diff
@@ -12,4 +14,41 @@ export async function debugExecAddAction(cmd: string, components: string[] = [])
...
+    writeFileSync(packagePath, JSON.stringify({
+      ...packageJson,
+      dependencies
+    }, null, 2))
+  }
+}
+
+export function debugRemovedPkg(components: string[], packagePath: string) {
+  if (!components.length) return
+  if (getStoreSync('debug')) {
```
Comment (wingkwong) [inline src/helpers/debug.ts:40]:
```ts
if (!components.length || !getStoreSync('debug')) return
```

File: src/actions/add-action.ts [hunk 3]
```diff
@@ -150,6 +150,11 @@ export async function addAction(components: string[], options: AddActionOptions)
     );
   }
 
+  if (getStoreSync('debug')) {
+    // Temporary add the components to the package.json file
```
Comment (wingkwong) [inline src/actions/add-action.ts:154]: grammar: 

```
Temporarily add the components to the package.json file
```

File: src/actions/add-action.ts [hunk 4]
```diff
@@ -238,5 +243,9 @@ export async function addAction(components: string[], options: AddActionOptions)
     )} whether in the correct place (ignore if added)\nSee more info here: ${DOCS_PROVIDER_SETUP}`
   );
 
+  if (getStoreSync('debug')) {
+    // Remove the temporary added components from the package.json file
```
Comment (wingkwong) [inline src/actions/add-action.ts:247]: grammar: 

```
Remove the temporarily added components from the package.json file
```

**Review by wingkwong — CHANGES_REQUESTED** [review]

**Review by wingkwong — COMMENTED** [review]
please check `src/actions/upgrade-action.ts` L41.

**Review by wingkwong — APPROVED** [review]

**Pull request merged**